src/browser/ui/dom/DOMChildrenOperations.js

```diff
@@ -31,10 +31,21 @@ function insertChildAt(parentNode, childNode, index) {
   // rely exclusively on `insertBefore(node, null)` instead of also using
   // `appendChild(node)`. However, using `undefined` is not allowed by all
   // browsers so we must replace it with `null`.
-  parentNode.insertBefore(
-    childNode,
-    parentNode.childNodes[index] || null
-  );
+
+  // fix render order error in safari
+  if (!(document.all && !document.addEventListener)) {
```

Comment:
IE8 use childNodes.item to read child is OK.But use it when dynamic insert will throw error, so it need detect ie 8.